integration-test.ps1

```diff
@@ -74,6 +98,7 @@ Write-TaskName "Verify Token"
 
 if ($LASTEXITCODE -ne 0) {
     Write-Output "Token did not verify (Error code $LASTEXITCODE)"
+    Write-Output "(You'll need to manually stop the server.)"
```

Comment:
To be nice to the user, we can instead do:

```powershell
$serverProcess = start-process powershell -PassThru -argument $command 
```

Then here:
```powershell
Stop-Process $serverProcess
```

Reply:
Oh wait, no we can't because that's a Powershell process and not the server...

Reply:
If we skipped using "sesclient.ps1" and invoke the exe directly then we can 😁

Reply:
I want to funnel through `sesclient.ps1` so that any smarts we include there are automatically used.